apps/passport-server/test/pretix/devconnectPretixDataMocker.ts

```diff
@@ -0,0 +1,174 @@
+import _ from "lodash";
+import { v4 as uuid } from "uuid";
+import {
+  DevconnectPretixOrder,
+  DevconnectPretixPosition,
+} from "../../src/apis/devconnectPretixAPI";
+import { logger } from "../../src/util/logger";
+
+export interface IMockDevconnectPretixData {
+  ordersByEventId: Map<string, DevconnectPretixOrder[]>;
+  eventNameByEventID: Map<string, string>;
+}
+
+export const EVENT_A = "event-a";
+export const EVENT_B = "event-b";
+export const EVENT_C = "event-c";
+
+export const EVENT_A_NAME = "Event A";
+export const EVENT_B_NAME = "Event B";
+export const EVENT_C_NAME = "Event C";
+
+export const ITEM_1 = 456;
+export const ITEM_2 = 123;
```

Comment:
should these be strings now?

Reply:
so they'll be returned as numbers from the API, so i just called toString() when necessary

not a perfect solution, perhaps we could do a `ITEM_1_ID_FROM_API` and `ITEM_1_ID_IN_DB` but that feels like overkill